FGEGraphics/UISystem/UIPositionHelper.cs

```diff
@@ -243,6 +262,13 @@ public UIPositionHelper GetterRotation(Func<float> rotation)
         return this;
     }
 
+    /// <summary>Fixes the position at the top-left origin.</summary>
+    /// <returns>This object.</returns>
+    public UIPositionHelper SetOrigin() => Anchor(UIAnchor.TOP_LEFT).ConstantXY(0, 0);
+
+    /// <summary>Returns a copy of the position fixed at the top-left origin.</summary>
+    public UIPositionHelper AtOrigin() => new UIPositionHelper(this).SetOrigin();
```

Comment:
What is the purpose of this? As opposed to just getting a new `UIPositionHelper.Origin` or whatever - what is intentionally copying from the prior?

Reply:
For composition. Take `UICheckbox` for example -- it's logical that the `UIPositionHelper` passed into the checkbox should control the width and height of the box itself as well as the position of the element. With composition, `UICheckbox` parents a `UIBox`; however, if we simply funnel the `pos` argument to that box, then the position of the box will be relative to the `UICheckbox`, not the *parent* of the checkbox. To achieve the desired effect, the box's position should be `pos` but at the origin.